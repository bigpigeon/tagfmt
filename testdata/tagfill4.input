//gofmt -f "json=big_camel(:field)"

package main

type OrderDetail struct {
    ID        string   ``
    User_Name string   ``
    OrderID   string   ``
    Callback  string   ``
    Address   []string ``
}
