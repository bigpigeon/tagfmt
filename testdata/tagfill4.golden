//tagfmt -f "json=upper_camel(:field)"

package main

type OrderDetail struct {
	ID        string   `json:"ID"`
	User_Name string   `json:"UserName"`
	OrderID   string   `json:"OrderID"`
	Callback  string   `json:"Callback"`
	Address   []string `json:"Address"`
}
